addons/core/functions/fnc_getSwSettings.sqf

```diff
@@ -65,6 +65,20 @@ if (isNil "_rc") then {
     _value set [TFAR_CODE_OFFSET, _rc];
 };
 
+private _hdo = _value select HALFDUPLEX_OVERRIDE_OFFSET;
```

Comment:
Agreed on `_hbo`, that could have definitely been named better.
`TFAR_fnc_getConfigProperty` unfortunately won't work since [it only works for `CfgVehicles`](https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/core/functions/fnc_getConfigProperty.sqf#L12), while a SR-Radio's config is stored in `CfgWeapons`. I think I was originally trying to use `TFAR_fnc_getConfigProperty`, but ran into this very issue and decided to deal with it the same way the rest of this file does.